front/pages/poke/[wId]/conversations/[cId]/index.tsx

```diff
@@ -0,0 +1,126 @@
+import { Page } from "@dust-tt/sparkle";
+import type {
+  AgentMessageType,
+  ContentFragmentType,
+  ConversationType,
+  UserMessageType,
+} from "@dust-tt/types";
+import { assertNever } from "@dust-tt/types";
+import type { InferGetServerSidePropsType } from "next";
+
+import PokeNavbar from "@app/components/poke/PokeNavbar";
+import { getConversation } from "@app/lib/api/assistant/conversation";
+import { withSuperUserAuthRequirements } from "@app/lib/iam/session";
+
+export const getServerSideProps = withSuperUserAuthRequirements<{
+  conversation: string;
+}>(async (context, auth) => {
+  const cId = context.params?.cId;
+  if (!cId || typeof cId !== "string") {
+    return {
+      notFound: true,
+    };
+  }
+
+  const conversation = await getConversation(auth, cId, true);
+  if (!conversation) {
+    return {
+      notFound: true,
+    };
+  }
+
+  return {
+    props: {
+      conversation: JSON.stringify(conversation),
```

Comment:
This is disgusting but getServerSideProps refuses our very complex type claiming it can't parse it to JSON

Didn't push further for now